gulpfile.js

```diff
@@ -77,150 +74,143 @@ gulp.task('serve', function () {
+    .pipe(sass({ outputStyle: "expanded" }).on("error", sass.logError))
+    .pipe(dest("docs/css/"));
+}
 
-gulp.task('sass', function () {
-  return gulp
-    .src('src/sass/**/*.scss')
-    .pipe(sass({ outputStyle: 'expanded' }).on('error', sass.logError))
-    .pipe(gulp.dest('docs/css/'));
-});
-
-gulp.task('sass:watch', function() {
-  gulp.watch('src/sass/**/*.scss', ['sass']);
-});
+function watchSass() {
+  watch("src/sass/**/*.scss", compileSass);
+}
 
 /**
  * Uses Rollup to compile ES6 down to browser JS with a UMD wrapper.
  * See more here:
  * https://rollupjs.org/guide/en#gulp
  */
-gulp.task('js', () => {
+function compileJS() {
   return rollup
-    .rollup({ input: './src/js/' + package.name + '.js', plugins: [babel()] })
+    .rollup({
+      input: "./src/js/" + package.name + ".js",
+      plugins: [babel()]
+    })
     .then(bundle => {
       return bundle.write({
-        file: './docs/js/' + package.name + '.js',
-        format: 'umd',
+        file: "./docs/js/" + package.name + ".js",
+        format: "umd",
         /**
          * Change this property to the namespace you want you're component
          * to have. For example "Widget". Then it's public methods should
          * be available as Widget.init().
          */
-        name: 'MyComponent',
+        name: "MyComponent",
         sourcemap: true
       });
-    })
-});
+    });
+}
 
-gulp.task('js:watch', function() {
-  gulp.watch('src/js/**/*.js', ['js']);
-});
+function watchJS() {
+  watch("src/js/**/*.js", compileJS);
+}
 
-gulp.task('js:copy', function() {
-  return gulp.src('./docs/js/**/*.js')
-    .pipe(gulp.dest('./dist/js/'));
-});
+function copyJS() {
+  return src("./docs/js/**/*.js").pipe(dest("./dist/js/"));
+}
 
-gulp.task('js:header', function() {
-  gulp.src('./dist/js/' + package.name + '.js')
+function headerJS(callback) {
+  src("./dist/js/" + package.name + ".js")
     .pipe(header(banner, { package: package }))
-    .pipe(gulp.dest('./dist/js/'));
+    .pipe(dest("./dist/js/"));
 
-  gulp.src('./dist/js/' + package.name + '.min.js')
+  src("./dist/js/" + package.name + ".min.js")
     .pipe(header(banner, { package: package }))
-    .pipe(gulp.dest('./dist/js/'));
-});
-
-gulp.task("js:minify", function(done) {
-  pump(
-    [
-      gulp.src('dist/js/' + package.name + '.js'),
-      uglify(),
-      rename({ suffix: '.min' }),
-      gulp.dest('dist/js')
-    ],
-    done
-  );
-});
-
-// Compiles, minifies, and versions JS
-gulp.task('js:release', function(done) {
-  runSequence(
-    'js',
-    'js:copy',
-    'js:minify',
-    'js:header',
-    done
-  );
-});
-
-gulp.task('css:copy', function() {
-  return gulp.src('./docs/css/**/*.css')
-    .pipe(gulp.dest('./dist/css/'));
-});
-
-gulp.task('css:minify', function () {
-  return gulp.src('dist/css/' + package.name + '.css')
+    .pipe(dest("./dist/js/"));
+
+  callback();
+}
+
+function minifyJS() {
+  return src("dist/js/" + package.name + ".js")
+    .pipe(
+      babel({
```

Comment:
I don't think this will do anything as that reference to `babel()` is actually the `rollup-plugin-babel` that we're importing up top. I'm pretty sure that outside fo the context of rollup it wouldn't work.